shell/common/gin_helper/event_emitter.h

```diff
@@ -62,6 +62,8 @@ class EventEmitter : public gin_helper::Wrappable<T> {
   // this.emit(name, new Event(flags), args...);
   template <typename... Args>
   bool EmitWithFlags(base::StringPiece name, int flags, Args&&... args) {
+    v8::Locker locker(isolate());
+    v8::HandleScope handle_scope(isolate());
```

Comment:
If you observed a crash related to this handle scope being missing, could you add a test which exercises the crashing behavior?

Reply:
This require user interaction with Tray and currently we don't have a way to test that.

I think it is obvious that `EmitWithFlags` requires a handle scope due to use of `internal::CreateEventFromFlags` so it should be fine without a test.